main.tf

```diff
@@ -77,7 +77,15 @@ locals {
   }
 
   node_pools_taints = {
-    all = []
+    all = [
+      for each in [
+        {
+          key    = "node.cilium.io/agent-not-ready"
+          value  = true
+          effect = "NO_EXECUTE"
```

Comment:
@EronWright Any difference by using `NoExecute` as the docs suggest?

Reply:
@maxsxu that's a good catch, and is weird, but is correct.  It's a GKE API thing.
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_cluster.html#nested_taint
https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/issues/152